src/ReadLine/project.json

```diff
@@ -26,6 +26,8 @@
         "NETStandard.Library": "1.6.1",
         "System.Console": "4.3.0"
       }
+    },
+    "net452": {
```

Comment:
Let's make this `net45` to support a broader range. Also the curly braces don't need to be on separate lines